packages/orbit-components/src/primitives/TooltipPrimitive/TooltipPrimitive.ct.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { test, expect } from "@playwright/experimental-ct-react";
+
+import type { Size } from "./types";
+import { DefaultTooltipPrimitive, TooltipPrimitiveWithImage } from "./TooltipPrimitive.ct-story";
+import type { Placement } from "../../common/placements";
+
+test.describe("Tooltip primitive visual tests", () => {
+  const placements = ["top", "right", "bottom", "left"];
+
+  placements.forEach(placement => {
+    test(`screenshot for default - ${placement}`, async ({ mount }) => {
+      const component = await mount(<DefaultTooltipPrimitive placement={placement as Placement} />);
+
+      await component.getByRole("button").hover();
```

Comment:
We can avoid having the hover action. This behavior should be tested in unit tests, not visual tests